openshift/templates/nodejs.json

```diff
@@ -157,6 +157,28 @@
                   {
                     "containerPort": 8080
                   }
+                ],
+                "env": [
```

Comment:
We need this to be able to connect to an existing MongoDB. We have already the parameters defined below in the template, and you can set them via Web Console, but then without this patch they are not passed to the application container.